src/systemTest/java/com/flightstats/hub/model/ZookeeperNode.java

```diff
@@ -0,0 +1,14 @@
+package com.flightstats.hub.model;
+
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.List;
+
+@Builder
+@Value
+public class ZookeeperNode {
+    Links _links;
+    ZookeeperNodeData data;
+    List<ZookeeperChildNode> children;
```

Comment:
In a effort to avoid null collections and try to ensure they're always just empty length collections, I've started doing this:
```
Links _links;
ZookeeperNodeData data;
@Builder.Default
List<ZookeeperChildNode> children = new ArrayList<>();
```
Figured I'd mention it in case you'd find it useful. I suppose it's generally useful, you could default a boolean to true for instance, but I generally find I want it for collections.

Reply:
Ooh, good idea. I remember having done some hacky things to avoid null collections in the past. This is nice and simple.